### PR TITLE
feat: Add cli command to generate cfast representation for cobol prog…

### PR DESCRIPTION
**Pull request by Nurkambay**
## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed a rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: CFAST.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+java -jar ./server/engine/target/server.jar cfast -ws $1
```
Comment (ishche) [inline CFAST.sh]: I think this file belongs to CCF repository.

File: server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliCFAST.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+ * Generates CFAST representation of the COBOL program in the defined folder
+ */
+@CommandLine.Command(name = "cfast", description = "generate cfast from cobol source")
+public class CliCFAST  implements Callable<Integer> {
+
+  @CommandLine.ParentCommand
+  private Cli parent;
+
+  @CommandLine.Option(
+      description = "Path to workspace folder.",
```
Comment (ishche) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliCFAST.java]: The naming is a bit confusing. Other commands assume that workspace is defined by processor groups configuration and related configuration, like set of programs, copybooks path, and  copybook extensions is taken from processor group configuration.
So let's rename the option to something like ccf-tests-folder, or change logic so it use processor groups.

**Review by ishche — CHANGES_REQUESTED** [review]

File: CFAST.sh (hunk 1 above)
Comment (Nurkambay) [reply CFAST.sh]: Ok

File: server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliCFAST.java (hunk 2 above)
Comment (Nurkambay) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliCFAST.java:48]: It is a copy/paste from a different command. I am just following code style and known name conventions

Comment (Nurkambay) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliCFAST.java]: This command is not CCF related thing, it is for generating CFAST. So I have renamed it to the source_folder

**Review by ishche — APPROVED** [review]
note: If this command is not specific to ccf, then it should support all related settings and processor groups in the future.

**Pull request merged**